### DISCUSSION TITLE
When I launch Atom using zsh, focus is returned to Terminal

### DISCUSSION THREAD

**Discussion by markwyner** [atom/atom#23270] [Q&A]
Hi, friends.

In my zsh file I have the following alias:

`alias zsh='atom ~/.zshrc'`

When I run "zsh" it opens the file in Atom, but returns the focus to Terminal. This isn’t a zsh, Terminal, or OS function (so far as I can tell) because if I run the same command for a different app, say, Sublime Text:

`alias zsh='subl ~/.zshrc'`

It opens in Sublime Text and stays there.

Any ideas why this might be happening?

**Comment by Tanapl02** [comment]
Thanks.
